gallery/README.md

```diff
@@ -0,0 +1,38 @@
+# Gallery
+
+Gallery is a script that automates process of applying different black versions to a
+selected PyPI package and seeing the results between black versions.
+
+## Build
+
+```console
+$ [sudo] docker build -t black_gallery .
```

Comment:
I suggest not putting `sudo` in the example, this is bad practice to run docker under root. https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user